front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorForm.tsx

```diff
@@ -179,6 +219,16 @@ const RollingStockEditorForm = ({
           count: invalidFields.length,
         })
       );
+    } else if (
+      isElectric(currentRsEffortCurve) &&
+      !(rollingStockValues.electricalPowerStartupTime && rollingStockValues.raisePantographTime)
+    ) {
+      dispatch(
+        setFailure({
+          name: t('messages.invalidForm'),
+          message: t('messages.missingElectricStartupAndPantoTime'),
+        })
+      );
```

Comment:
I would prefer that everything is handle in checkRollingStockFormValidity. Could move the validation there, and also adapt the invalidFields list (add electricalPowerStartupTime and raisePantographTime) ?